build_library/grub_install.sh

```diff
@@ -110,6 +116,21 @@ info "Installing GRUB ${FLAGS_target} in ${FLAGS_disk_image##*/}"
 LOOP_DEV=$(sudo losetup --find --show --partscan "${FLAGS_disk_image}")
 ESP_DIR=$(mktemp --directory)
 
+# Hack available for doing the build in a container; see the LOOP_NO_UDEV
+# comments above.
+if [[ ${LOOP_NO_UDEV} -eq 1 ]]; then
+    echo "LOOP_NO_UDEV enabled, creating device nodes directly"
+    LOOPNUM=$(echo ${LOOP_DEV} | sed -e 's,/dev/loop,,')
+    for d in /sys/block/loop${LOOPNUM}/loop${LOOPNUM}p*; do
```

Comment:
For additional cleanliness, maybe have a `shopt -s nullglob` for this loop?  That way if something crazy happens that breaks this pattern, the rest of the script still functions normally when it checks for a nonexistent device node.  (This "library" script is only executed standalone rather than sourced, so a `shopt` shouldn't affect anything else.)